VimeoNetworking/VimeoNetworking/Models/VIMAccount.m

```diff
@@ -94,9 +91,9 @@ - (id)initWithCoder:(NSCoder *)aDecoder
 - (void)encodeWithCoder:(NSCoder *)aCoder
 {
     [aCoder encodeObject:self.accessToken forKey:NSStringFromSelector(@selector(accessToken))];
-    [aCoder encodeObject:self.tokenType forKey:NSStringFromSelector(@selector(tokenType))];
-    [aCoder encodeObject:self.scope forKey:NSStringFromSelector(@selector(scope))];
-    [aCoder encodeObject:self.userJSON forKey:NSStringFromSelector(@selector(userJSON))];
+    [aCoder encodeObject:self.tokenType forKey:@"tokenType"];
```

Comment:
@alfiehanssen Good catch, mustve taken the wrong version for this file. Fixing..